packages/kbn-ftr-common-functional-ui-services/services/security/role.ts

```diff
@@ -23,6 +28,7 @@ export class Role {
         kibana: role.kibana,
         elasticsearch: role.elasticsearch,
       },
+      headers: HEADERS,
```

Comment:
Not quite sure why the version header was needed with the axios request. I can successfully issue the request via curl or postman without the version header, but the axios request was receiving status 400 without it.

Reply:
As discussed in slack, I think we can move on with this for now, but should investigate this more.

Reply:
The config that was failing was `x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/shared/config.ts`
Maybe it has to do with having SSL enabled for ES?






[3:33](https://elastic.slack.com/archives/D047YJN5KE2/p1727444030619969)
Maybe it has to do with the SSL config?

Reply:
Resolved in 5e42731df1a9b2330ac0695f4501211e266a002d. Thanks, JL!